### ISSUE TITLE
Input Echo Hub...

### ISSUE THREAD

**Issue by nitrohaiza** [datech/node-red-contrib-amazon-echo#98]
Hi there,

I have a question. How can I control a device from the outside? I set process and output at the hub. That means I would send my request at the entrance to the hub. Is that correct?

Thanks a lot.

**Comment by nitrohaiza** [comment]
<img width="375" alt="Function" src="https://user-images.githubusercontent.com/60295215/73124084-4f397a80-3f97-11ea-8d5c-91849f85b7e7.png">
<img width="968" alt="Komplett" src="https://user-images.githubusercontent.com/60295215/73124085-4f397a80-3f97-11ea-893a-4f0923eaab81.png">


Can anyone tell me, what i must write? 

Sorry my language is German. Not so easy for me.

Phil

**Comment by flowerbob** [comment]
I am having the same issue. Can´t get the Hub recognize the input message. 
"Process Input" setting is set to "process" 
Here is my try of the function in front of the hub (same setup as yours):
![Unbenannt](https://user-images.githubusercontent.com/53487904/73601609-d7ee8280-4564-11ea-9730-de8b09e8a2e6.PNG)

**Comment by datech** [comment]
Related to #90

**Comment by datech** [comment]
@nitrohaiza / @flowerbob, did you manage to resolve it?

**Comment by flowerbob** [comment]
Yes
But for my purpose I still need the rgb value input for the hub so right now only brightness and state are fed into the hub.
I take the message coming from home assistant (current state node) and feed it trough a function containing code into the hub.

**function code:**
``` {
var bri =msg.data.attributes.brightness;
var name =msg.data.attributes.friendly_name;
var state = msg.data.state;
var onstate;
if (state == "on") {
   onstate= true;
} else {
   onstate= false;
}
var Msg = {
    "nodename": name,
    "bri" : bri,
    "on" : onstate,
}
var newMsg = { payload: Msg };
return newMsg;
}```

**Comment by datech** [comment]
RGB input is implemented as part of #15, but it is still not released, so it has to be available in the next version.

**Comment by datech** [comment]
RGB input support has been released in v0.1.10

**Comment by fercod** [comment]
I had the same problem, I hope this can help someone:

![image](https://user-images.githubusercontent.com/82311158/181919453-f77ea715-8d08-41a7-b5e7-bf1db4f3c8a4.png)
![image](https://user-images.githubusercontent.com/82311158/181919474-68726706-e6d9-4de0-bdfe-1474057a9a00.png)
![image](https://user-images.githubusercontent.com/82311158/181919523-2a418bbd-0630-4762-ada8-054cf8392103.png)
![image](https://user-images.githubusercontent.com/82311158/181919532-e9d5d6df-485c-41fc-bf02-11b9dba7510b.png)
